datahub-web-react/src/app/lineage/LineageEntityNode.tsx

```diff
@@ -90,12 +91,17 @@ export default function LineageEntityNode({
     const [isExpanding, setIsExpanding] = useState(false);
     const [expandHover, setExpandHover] = useState(false);
     const { getAsyncEntity, asyncData } = useLazyGetEntityQuery();
```

Comment:
well we need both right now because the getAsyncLineage query only gets lineage data and we need this query to get all the other info about entities (like `name`, `platform`, etc.) - however I could update the lineage query to get everything we need and then we only have the one query. 

Will do now